deployer/jsonrpc-deployer/jsonrpc-private-key-deployer.js

```diff
@@ -16,15 +17,21 @@ class JSONRPCDeployer extends PrivateKeyDeployer {
 	constructor(privateKey, nodeUrl, defaultOverrides) {
 		const localNodeProvider = new ethers.providers.JsonRpcProvider(nodeUrl, ethers.providers.networks.unspecified);
 		super(privateKey, localNodeProvider, defaultOverrides);
+		this._validation(nodeUrl);
 		this.nodeUrl = nodeUrl;
 		console.log(`Network: ${colors.colorNetwork(this.nodeUrl)}`)
-
 	}
 
 	toString() {
 		const superString = super.toString();
 		return `Network: ${colors.colorNetwork(this.nodeUrl)}\n${superString}`;
 	}
+
+	_validation(nodeUrl) {
+		if (!(isUrl(nodeUrl))) {
+			throw new Error(`Passed contract url (${isUrl}) is not valid url`);
```

Comment:
You have not tested this. `isUrl` is function not a variable, you need to pass  `nodeUrl` as param to the string literal